### PR TITLE
Upgrade SBT major version

### PR DESCRIPTION
**Pull request by houcros**

### CODE REVIEW

**Comment by houcros** [comment]
@simao should I tag this commit?

File: keyserver/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.2.8
```
Comment (jochenschneider) [inline keyserver/project/build.properties]: Stupid question: why is the SBT version defined in each sub-project? Can we just leave that out?

Comment (houcros) [reply keyserver/project/build.properties]: Good point. I tried removing the ones in the subprojects and it doesn't seem to make a difference. So I can amend the commit and remove them, unless @simao you think there's a reason to keep them?

Comment (simao) [reply keyserver/project/build.properties]: Lets use just one if that works.

**Pull request merged**